WordPress/gutenberg
# Update Navigation block placeholder

## Description

Switch larger standard placeholder to inline menu that fits in more places. This updates the overall placeholder when the block is first inserted. There is a related PR here that updates the empty state when a menu is inserted #26947 

Fixes #23207 

## How has this been tested?

- Add Navigation block both horizontal and vertical
- Add with/without existing Pages
- Add with/without existing Menus
- Confirm adding Pages, Menus, and Links work

## Types of changes

- Switches away from standard Placeholder
- Moves actions from one menu to multiple inline actions

![navigation-placeholder](https://user-images.githubusercontent.com/45363/99314410-ed901880-2815-11eb-945c-a408924ae1a4.gif)


## TODO

- [X] One issue to fix still: the horizontal and vertical type isn't working in editor, but it does work for the menu.
